recipes/qtconsole/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+xorg-x11-server-Xvfb
```

Comment:
This how we install things that are not included in the docker container. It is just a file with each requirement on a newline. This just gets fed directly into `yum`. Also, with recent changes in `conda-smithy` this will be handled on feedstock conversion.